cfg/config.yaml

```diff
@@ -173,3 +173,17 @@ version_mapping:
   "1.17": "cis-1.5"
   "ocp-3.10": "rh-0.7"
   "ocp-3.11": "rh-0.7"
+
+target_mapping:
+  "cis-1.3":
+    - "master"
+    - "node"
+  "cis-1.4":
+    - "master"
+    - "node"
+  "cis-1.5":
+    - "master"
+    - "node"
+    - "controlplane"
+    - "etcd"
+    - "policies"
```

Comment:
I think there also needs to be an entry for "rh-0.7" which lists "master" and "node"

Reply:
Hi @leodotcloud, if I'm not mistaken it's just this missing `rh-0.7` entry that's standing between us merging this PR, would you be OK to add that?

Reply:
Sorry, have been busy. Will try to rework on this PR early next week.